### ISSUE TITLE
Select multiple orders using existing checkboxes, and perform key actions from Actions dropdown

### ISSUE THREAD

**Issue by kirstenalarsen** [openfoodfoundation/openfoodnetwork#4436]
## What is the problem we are solving 
The Order Management page now checkboxes where multiple orders can be selected. However the only function that can then be activated is the 'bulk invoice printing'

This is counter-intuitive and out of step with user expectations - key functions like cancelling orders, resending order confirmations etc could be activated from here.

There is already a box with a range of actions that can be taken on an order, accessible once you have opened a single order for editing

Can we just make this same box available on the order management page (replacing the Print Invoices single button), and enable it to be used on multiple orders that have been selected with checkboxes?

![image.png](https://images.zenhubusercontent.com/54b83fb37c6ccc6277a6ea46/99550587-20de-4ffc-95ba-e93b6388850e)

## Success factors = expected outcome 
I can select multiple orders using checkboxes, and perform key order actions (as per existing Actions dropdown with the order edit) on all of them with one click

NB. If this were accepted as a papercut, it would potentially supercede the preferred implementation of capture payments in #4377, as it would then make sense to just include Capture Payments within the Actions dropdown

**Comment by luisramos0** [comment]
I'd say this is S+, we need to build a new api endpoint to make each bulk action happen.
From discourse I see papercuts are XS :-(

**Comment by lin-d-hop** [comment]
In that case shall we just close this @kirstenalarsen and stick with #4377 ?

**Comment by kirstenalarsen** [comment]
yes I think so @lin-d-hop . Bit curious about process - like should we completely close this or just move it to 'rejected papercuts' but leave in all the things? It would still be a good thing to do, and could perhaps be a reasonable new dev task - is that right @luisramos0 or would be too hard?

**Comment by luisramos0** [comment]
it's not too hard but it's not an good-first-issue

**Comment by Erioldoesdesign** [comment]
Design additional consideration for this: If a user is performing many actions across orders say...100 orders are sending the reconfirmation email action.

I think we may need to consider a modal, message or some UI that indicates how long the system will take to complete it *if it is not done within 10 seconds* 

In my past org we have a dialog that pops up when performing bulk actions that might take a long time here it is:

<img width="995" alt="Screenshot 2020-11-02 at 22 39 21" src="https://user-images.githubusercontent.com/11681324/97927134-511d3f00-1d5c-11eb-9177-bd9a56a7c697.png">
